source/backend/api/Helpers/Extensions/DocumentUploadRequestExtension.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using Pims.Api.Models.Requests.Document.Upload;
+using Pims.Core.Api.Exceptions;
+
+namespace Pims.Api.Helpers.Extensions
+{
+    public static class DocumentUploadRequestExtension
+    {
+        public static void ThrowInvalidFileSize(this DocumentUploadRequest documentUploadRequest)
```

Comment:
Hmm, I'm curious about what the benefit of moving this to an extension method is, can you provide some background?

Reply:
this private method was first defined on a different Service and since I needed it on another it made sense to put it as an Extension method at that time. At this point if it's only used in one service I could make it back to a private method.